proseco/characteristics.py

```diff
@@ -42,56 +41,11 @@
 bad_pixels = [[-245, 0, 454, 454],  # Bad column
               [-374, -374, 347, 347]]  # Fid trap
 
-bad_star_set = set([36178592,
```

Comment:
And I *think* we had these in here before we started syncing the supplement (and as a resource for testing) so these can all go... but I don't remember if there was any other reason for the hardcoded list.